bioio_ome_zarr/writers/ome_zarr_writer_v3.py

```diff
@@ -0,0 +1,497 @@
+import sys
+
+if sys.version_info < (3, 11):
+
+    class OMEZarrWriter:
```

Comment:
Should this just fail on import for < 3.11 rather than failing on instantiation?